src/theme/components/checkbox.ts

```diff
@@ -36,8 +36,11 @@ const baseStyleControl = {
 
     _disabled: {
       bg: "ui.white",
-      borderColor: "ui.gray.medium",
+      borderColor: "ui.disabled.primary",
       color: "ui.gray.medium",
```

Comment:
Some of the other styles have `"ui.gray.dark"`. Should this be changed?

Reply:
Actually, this style assignment doesn’t do anything.  The label color value is coming in through `checkboxRadioLabelStyles` from the globals file.  I am going to remove it altogther.